Assets/Scripts/Effects/AnimateWater.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Effects
+{
+    public class AnimateWater : MonoBehaviour
+    {
+        [SerializeField] [Tooltip("Determines the difference between the peak and bottom of a wave")]
+        private float power = 3f;
+    
+        [SerializeField] [Tooltip("Determines the size of waves")]
+        private float scale = 1f;
+    
+        [SerializeField] [Tooltip("Determines how fast waves aree")]
+        private float frequency = 1f;
+
+        private float _xOffset, _yOffset;
+        private MeshFilter _mf;
+
+
+        void Start()
+        {
+            _mf = GetComponent<MeshFilter>();
+            AnimateWaves();
```

Comment:
Is there a reason why you call this in Start already? The call in Update should be enough

Reply:
It starts the animation at frame 0 i guess. it's not particularly necessary, since you wont be able to notice it if it's not the. I can delete this.